src/mysql_vm_helpers.py

```diff
@@ -107,6 +96,17 @@ def install_and_configure_mysql_dependencies() -> None:
             snap.SnapNotFOundError, snap.SnapError: if issue installing mysql shell snap
         """
         try:
+            # install the charmed-mysql snap
+            logger.debug("Retrieving snap cache")
+            cache = snap.SnapCache()
+            charmed_mysql = cache[CHARMED_MYSQL_SNAP_NAME]
+
+            if not charmed_mysql.present:
+                logger.debug("Installing charmed-mysql snap")
+                charmed_mysql.ensure(
+                    snap.SnapState.Latest, channel="8.0/edge/whatever-shayan-wants"
```

Comment:
Reminder that this is a temp branch name :)

Reply:
changed in [10cac4e](https://github.com/canonical/mysql-operator/pull/128/commits/10cac4e53675e51e253ce9d65117e55e89f7b095)